sdk/src/signer/keypair.rs

```diff
@@ -1,387 +1,4 @@
-#![cfg(feature = "full")]
-
-#[cfg(target_arch = "wasm32")]
-use wasm_bindgen::prelude::*;
-use {
-    crate::{
-        pubkey::Pubkey,
-        signature::Signature,
-        signer::{EncodableKey, EncodableKeypair, SeedDerivable, Signer, SignerError},
-    },
-    ed25519_dalek::Signer as DalekSigner,
-    ed25519_dalek_bip32::Error as Bip32Error,
-    hmac::Hmac,
-    rand0_7::{rngs::OsRng, CryptoRng, RngCore},
-    solana_derivation_path::DerivationPath,
-    std::{
-        error,
-        io::{Read, Write},
-        path::Path,
-    },
+pub use {
```

Comment:
Do we need the deprecation message here?

Reply:
Done with [06426cc](https://github.com/anza-xyz/agave/pull/3087/commits/06426cc53236039eca9aac6569d6de62c1a569d2)